colors.css

```diff
@@ -0,0 +1,19 @@
+/* Light theme */
```

Comment:
Any suggestions? `whitespace-colors-template.css`? What about moving this and the LESS to a styles/ folder, seeing that we have two stylesheets now?